redis-instance/values.yaml

```diff
@@ -42,10 +42,15 @@ master:
     enabled: true
     ## @param master.persistence.medium Provide a medium for `emptyDir` volumes.
     ##
+    path: /data
```

Comment:
I take it this is the default value and we're just making it explicit so it's clear it has to match the below?

Reply:
yes exactly, sorry I should have made that clear.